study-builder/studies/brugada/study-workflows.conf

```diff
@@ -0,0 +1,95 @@
+{
+  "workflowTransitions": [
+    {
+      "from": {
+        "type": "START"
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "PREQUAL",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "PREQUAL"
+      },
+      "to": [
+        {
+          "type": "DASHBOARD",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "CONSENT"
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "MEDICAL_RECORD_RELEASE",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "MEDICAL_RECORD_RELEASE"
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "BACKGROUND_INFORMATION",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "BACKGROUND_INFORMATION"
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "GENETIC_TESTING",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "GENETIC_TESTING"
+      },
+      "to": [
+        {
+          "type": "DASHBOARD",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "RETURN_USER"
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "CONSENT",
+          "expression": """!user.studies["brugada"].forms["CONSENT"].hasInstance()"""
```

Comment:
Looks like workflow is NOT correct for "RETURN_USER"
If user never submitted "CONSENT" , RETURN_USER should goto CONSENT rather than dashboard.
If RETURN_USER submitted CONSENT, user should be given next in-progress (NOT yet submitted) activity. 
If ALL activities are submitted, RETURN_USER should goto DASHBOARD.
Please look at existing study RETURN_USER workflow and change accordingly.

Reply:
Once changes are completed and study is redeployed in "Dev" RETURN_USER workflow should be tested for above scenarios.

Reply:
I know this approach is used in some of the studies. However, it's not the only approach. Here we redirect the user to the dashboard if consent has been created.

Reply:
If the requirment it to goto DASHBOARD, thats fine.:+1: